test/__tests__/typedef-with-template-test.ts

```diff
@@ -29,4 +29,9 @@ describe("Typedef with template", () => {
     const text = Utils.replaceVersionInfo(generateCode);
     expect(text).toMatchSnapshot();
   });
+  test("remote-ref-access", () => {
+    const generateCode = fs.readFileSync(path.join(__dirname, "../code/typedef-with-template/remote-ref-access.ts"), { encoding: "utf-8" });
```

Comment:
```ts
import { posix as path } from "path";
```

Oh, sorry. Please change import `path`. Like this.

https://github.com/Himenon/openapi-typescript-code-generator/blob/main/scripts/tools/cherry-pick.ts#L2

Reply:
path" does not work properly in Windows environment, use posix path.

https://nodejs.org/api/path.html#pathposix